src/libraries/ExcelConverter/Utils.cs

```diff
@@ -252,6 +254,19 @@ public static String ReformatRange(String input)
             return colonRangeRegex.Replace(input, "$1_RANGE_$2");
         }
 
+        // Takes in a range eg. "A4:C9" and returns its corresponding rang eobject
+        public static Range DecomposeRange(String strInput)
```

Comment:
Do you think it would be valuable to have the Range object take a string in a constructor, so you can create a range directly from a string like: new Range(stringVar)

Reply:
will resolve later during refactoring phase